src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterIdFormat.html

```diff
@@ -0,0 +1,40 @@
+<!--
```

Comment:
Can we use java comment?

Reply:
I will try to.

Reply:
We can but it is not officially recognized to use `/* ... */` in `html` files. File editors also would not recognize it.

Reply:
Rename file to be *.htmltxt to calm down your IDE

Reply:
Shall I do that for other files as well then?

Reply:
The reason to request all these java comments is to be able to use `verifyFilterWithInlineConfigParser` instead of `verify`, am I getting what you mean?

Reply:
Done. Also did that in other files - `xml`, `yaml`, etc. Removed all usages of `verify()` and swapped them with `verifyFilterWithInlineConfigParser()`. Also removed suppression for `forbidden-apis` since `verify` is no longer used.